src/components/organisms/BannerAdmin/BannerAdmin.tsx

```diff
@@ -1,89 +1,190 @@
-import React, { useState, useCallback, useRef } from "react";
+import React, { useCallback } from "react";
+import { useForm } from "react-hook-form";
+import classNames from "classnames";
 
 import { makeUpdateBanner } from "api/bannerAdmin";
 
 import { AnyVenue } from "types/venues";
+import { Banner } from "types/banner";
+
+import { useShowHide } from "hooks/useShowHide";
+
+import { Button } from "components/atoms/Button";
+import { Checkbox } from "components/atoms/Checkbox";
+import { InputField } from "components/atoms/InputField";
+import { ConfirmationModal } from "components/atoms/ConfirmationModal/ConfirmationModal";
+
+import "./BannerAdmin.scss";
+import { useAsyncFn } from "react-use";
```

Comment:
You should move this at the top with the other module imports.

Reply:
fixed 42fc0f3